### ISSUE TITLE
Window install is not working (Failed at the react-native-interactable@0.1.10 postinstall script.) 

### ISSUE THREAD

**Issue by bbeckk** [wix/react-native-interactable#210]
Nothing seems to be working:
1)Using gitbash and cmd has exact same errors
2)Updating npm doesn't work

```
E:\zreactNative\test\CollapsibleFilterTest>npm install react-native-interactable --save

> react-native-interactable@0.1.10 postinstall E:\zreactNative\test\CollapsibleFilterTest\node_modules\react-native-interactable
> ln -sf lib/{ios,android} .

ln: creating symbolic link `./ios' to `lib/ios': Permission denied
ln: creating symbolic link `./android' to `lib/android': Permission denied
npm WARN eslint-plugin-react-native@3.2.1 requires a peer of eslint@^3.17.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-native-interactable@0.1.10 postinstall: `ln -sf lib/{ios,android} .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-native-interactable@0.1.10 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\zzz\AppData\Roaming\npm-cache\_logs\2018-05-03T05_45_55_628Z-debug.log
```

**Comment by pkmusthafa** [comment]
Facing the same issue with git bash while executing npm install

$ npm install react-native-interactable@0.1.10

> react-native-interactable@0.1.10 postinstall d:\ReactNative\TestApp\node_module
s\react-native-interactable
> ln -sf lib/{ios,android} .

ln: creating symbolic link `./ios' to `lib/ios': Permission denied
ln: creating symbolic link `./android' to `lib/android': Permission denied
npm WARN react-native-interactable@0.1.10 requires a peer of react@>= 15.4.1 but
 none is installed. You must install peer dependencies yourself.
npm WARN react-native-interactable@0.1.10 requires a peer of react-native@>= 0.4
0.0 but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-native-interactable@0.1.10 postinstall: `ln -sf lib/{ios,android}
 .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-native-interactable@0.1.10 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.

**Comment by bbeckk** [comment]
Updated the git bash and it worked. I had a git version Git-1.9.4 installed previously and now I had updated it to Git-2.17.0-64-bit and it worked.   https://git-scm.com/download/win

**Comment by zengyimou** [comment]
Facing the same issue with git bash while executing npm install

**Comment by st-sloth** [comment]
This issue is fixed in #211. Would be nice to get it merged and stop this torrent of installing problems.

**Comment by traxx10** [comment]
Am also getting the same error
